documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2179,17 +2179,21 @@ The following _advanced_ configuration properties have defaults that work in mos
 |`1000`
 |Positive integer value that specifies the number of milliseconds the connector should wait for new change events to appear before it starts processing a batch of events. Defaults to 1000 milliseconds, or 1 second.
 
-|[[db2-property-max-queue-size]]<<db2-property-max-queue-size, `+max.queue.size+`>>
-|`8192`
-|Positive integer value for the maximum size of the blocking queue. The connector places change events that it reads from the database log into the blocking queue before writing them to Kafka. This queue can provide backpressure for reading change-data tables when, for example, writing records to Kafka is slower than it should be or Kafka is not available. Events that appear in the queue are not included in the offsets that are  periodically recorded by the connector. The `max.queue.size` value should always be larger than the value of the `max.batch.size` connector  configuration property.
-
 |[[db2-property-max-batch-size]]<<db2-property-max-batch-size, `+max.batch.size+`>>
 |`2048`
 |Positive integer value that specifies the maximum size of each batch of events that the connector processes.
 
+|[[db2-property-max-queue-size]]<<db2-property-max-queue-size, `+max.queue.size+`>>
+|`8192`
+|Positive integer value for the maximum size of the blocking queue. The connector places change events that it reads from the database log into the blocking queue before writing them to Kafka. This queue can provide backpressure for reading change-data tables when, for example, writing records to Kafka is slower than it should be or Kafka is not available. Events that appear in the queue are not included in the offsets that are  periodically recorded by the connector. The `max.queue.size` value should always be larger than the value of the `max.batch.size` connector  configuration property.
```

Comment:
This text differes form other connectors. Could it be unified?

Reply:
@jpechane I normalized the description across the different connectors. I left alone the description of the property in the  Cassandra doc, because it contained some unique information. If you approve of the revised text and you believe that the same description can be applied for Cassandra. let me know and I'll update it there as well.